features/rni.feature

```diff
@@ -0,0 +1,204 @@
+# language: nl
+
+Functionaliteit: RNI-deelnemer voor geleverde gegevens
+
+
+  Rule: de RNI-deelnemer gegevens worden geleverd voor elke categorie waarvan ten minste één gegeven gevraagd is
+    - RNI-deelnemer gegevens worden ook geleverd wanneer hier niet naar gevraagd is met de fields parameter
+    - categorie wordt gevuld met de naam van categorie waarop de RNI-deelnemer gegevens heeft aangeleverd
+      | categorie | naam           |
+      | 1         | Persoon        |
+      | 4         | Nationaliteit  |
+      | 6         | Overlijden     |
+      | 7         | Inschrijving   |
+      | 8         | Verblijfplaats |
```

Comment:
categorieën worden in het LO met een voorloopnul beschreven ( 01,04, etc...)

Reply:
aangepast